connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewContainerController.kt

```diff
@@ -177,9 +237,14 @@ internal class StripeConnectWebViewContainerController<Listener : StripeEmbedded
         if (permissionsRequested.isEmpty()) { // all calls to PermissionRequest must be on the main thread
             withContext(Dispatchers.Main) {
                 request.deny() // no supported permissions were requested, so reject the request
-                // TODO - add an analytic event here to track this unexpected behavior
+                analyticsService.track(
+                    ConnectAnalyticsEvent.ClientError(
+                        error = "Unexpected permissions request",
+                        errorMessage = "Unexpected permissions '${request.resources.joinToString()}' requested"
```

Comment:
Is it possible that `request.resources` may contain PII? If so, we shouldn't include it in the analytic.